src/converse-controlbox.js

```diff
@@ -7,33 +7,33 @@
 /**
  * @module converse-controlbox
  */
-import "converse-chatview";
-import "converse-profile";
-import "converse-rosterview";
-import _FormData from "formdata-polyfill";
-import bootstrap from "bootstrap.native";
-import converse from "@converse/headless/converse-core";
-import tpl_brand_heading from "templates/converse_brand_heading.html";
-import tpl_controlbox from "templates/controlbox.html";
-import tpl_controlbox_toggle from "templates/controlbox_toggle.html";
-import tpl_login_panel from "templates/login_panel.html";
+import 'converse-chatview';
+import 'converse-profile';
+import 'converse-rosterview';
+import _FormData from 'formdata-polyfill';
+import bootstrap from 'bootstrap.native';
+import converse from '@converse/headless/converse-core';
+import tpl_brand_heading from 'templates/converse_brand_heading.html';
+import tpl_controlbox from 'templates/controlbox.html';
+import tpl_controlbox_toggle from 'templates/controlbox_toggle.html';
+import tpl_login_panel from 'templates/login_panel.html';
```

Comment:
Looks like you ran this file through a code formatter (prettier?) which makes it impossible to see what changes you actually made to it.

Please add an exception for this repo to your code formatter.